packages/core/rollup.config.js

```diff
@@ -119,6 +121,16 @@ export default {
             ? generateCssModulesMockName(name)
             : generateCssModulesScopedName(name, filename, css)
       }
+    }),
+    copy({
+      targets: [
+        { src: "src/constants/colors.json", dest: "dist/assets" },
+        {
+          src: "../../node_modules/monday-ui-style/dist/index.min.css",
+          dest: "dist/tokens",
+          rename: () => "tokens.css"
```

Comment:
we should definitely consider - already in Vibe3 - to remove the tokens from monday-ui-react-core
wdyt?

Reply:
(not necessarily in this PR)